pom.xml

```diff
@@ -31,6 +31,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <ebean.agent.version>10.1.2</ebean.agent.version>
```

Comment:
Ebean version is not necessary the same as the Ebean agent version. Could you create `ebean.version` variable set to _10.1.3_ please